Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -15,6 +15,39 @@
 require "formula"
 
 module Homebrew
+  def inreplace_pairs(formula, replacement_pairs)
```

Comment:
It would feel more appropriate if the first argument were a `Pathname` instance. This method doesn't need to know that it is operating on a formula (and it could as well operate on an arbitrary file).